src/main/java/org/myrobotlab/framework/Plan.java

```diff
@@ -21,13 +29,9 @@ public class Plan {
 
   LinkedHashMap<String, ServiceConfig> config = new LinkedHashMap<>();
```

Comment:
Always agree with "variable declarations use the interface type and not a concrete" ... this is just sloppyness 
fixed.  If there is a question on access, I typically go with "protected" as I know this will allow for serialization yet be restrictive as possible, but still allow overrideing if necessary by a child class